verilog/analysis/verilog_linter_configuration.h

```diff
@@ -130,6 +130,30 @@ struct ProjectPolicy {
   std::string ListPathGlobs() const;
 };
 
+struct LinterConfigurationFlags {
```

Comment:
This name sounds very specific to be tied to flags. But in general this is more like a configuration struct, which we commonly would call somethinsomethingOptions; `LinterOptions` maybe ? ('Configuration' could be added but would not add more to the meaning).

Reply:
Sure, makes sense, renamed